microstoragetest/storage.go

```diff
@@ -0,0 +1,18 @@
+package microstoragetest
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/microstorage"
+	"github.com/giantswarm/microstorage/memory"
+)
+
+// New creates a configured storage ready to be used in tests.
+func New() microstorage.Storage {
```

Comment:
IMO `Must` is appropriate in case the method can panic and does not have error handling. This code here can panic because it uses `panic`. Doesn't matter if it has input or not. Things can change at runtime, even though not really in this specific implementation though, but when it cannot panic then don't use `panic`. Otherwise use the naming with `Must`.